src/test/java/uk/gov/justice/hmpps/nomis/datacompliance/controller/DataComplianceControllerTest.java

```diff
@@ -3,27 +3,34 @@
 import net.syscon.elite.api.model.PendingDeletionRequest;
 import net.syscon.elite.api.resource.impl.ResourceTest;
 import org.junit.Test;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import uk.gov.justice.hmpps.nomis.datacompliance.service.OffenderDataComplianceService;
-
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.dto.OffenderPendingDeletionEvent;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.dto.OffenderPendingDeletionEvent.Booking;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.dto.OffenderPendingDeletionEvent.OffenderWithBookings;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.dto.OffenderPendingDeletionReferralCompleteEvent;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.publishers.OffenderPendingDeletionEventPusher;
+
+import java.time.LocalDate;
 import java.time.LocalDateTime;
-import java.util.concurrent.CompletableFuture;
 
 import static net.syscon.elite.executablespecification.steps.AuthTokenHelper.AuthToken.ELITE2_API_USER;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.eclipse.jetty.http.HttpStatus.ACCEPTED_202;
 import static org.eclipse.jetty.http.HttpStatus.BAD_REQUEST_400;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
 import static org.springframework.http.HttpMethod.POST;
 
 public class DataComplianceControllerTest extends ResourceTest {
 
     private static final String REQUEST_ID = "123";
-    private static final LocalDateTime WINDOW_START = LocalDateTime.now();
-    private static final LocalDateTime WINDOW_END = WINDOW_START.plusDays(1);
 
-    @MockBean
-    private OffenderDataComplianceService offenderDataComplianceService;
+    // This date is 7 years after the SED_CALCULATED_DATE of the expected record
+    private static final LocalDateTime WINDOW_START = LocalDateTime.of(2027, 3, 24, 0, 0);
```

Comment:
You could call the variable something that indicates its purpose... like WINDOW_START_AFTER_DELETION_PERIOD.  And I guess that number 7 is defined somewhere?  You could use that when building the date.